src/core_modules/capture-core/hooks/useScopeTitleText.js

```diff
@@ -7,7 +8,8 @@ export const useScopeTitleText = (scopeId: ?string) => {
 
     const text = {
         [scopeTypes.EVENT_PROGRAM]: `${programName}`,
-        [scopeTypes.TRACKER_PROGRAM]: `${trackedEntityName} in program: ${programName}`,
+        [scopeTypes.TRACKER_PROGRAM]:
+            `${i18n.t('{{trackedEntityName}} in program', { trackedEntityName })}: ${programName}`,
         [scopeTypes.TRACKED_ENTITY_TYPE]: `${trackedEntityName}`,
```

Comment:
Actually, I think I would change this entire object to: 

```
      const text = {
        [scopeTypes.EVENT_PROGRAM]: programName,
        [scopeTypes.TRACKER_PROGRAM]:
            i18n.t('{{trackedEntityName}} in program: {{programName}}', {
                trackedEntityName,
                programName,
            }),
        [scopeTypes.TRACKED_ENTITY_TYPE]: trackedEntityName,
    };
```

This will change even more if you apply the suggestions with the extra param in the function declaration